app/controllers/application_controller.rb

```diff
@@ -33,12 +33,20 @@ def guest_user(with_retry = true)
   # called (once) when the user logs in, insert any code your application needs
   # to hand off from guest_user to current_user.
   def logging_in
-    # For example:
-    # guest_comments = guest_user.comments.all
-    # guest_comments.each do |comment|
-      # comment.user_id = current_user.id
-      # comment.save!
-    # end
+    #Transfer guest unit, team, and bills to the new user account
+    current_user.unit_id = guest_user.unit_id
+    current_user.team_id = guest_user.team_id
+    current_user.save
```

Comment:
You _should_ be able do something like:

    current_user.update(unit: guest_user.unit, team: guest_user.team)

Thoughts?